wagtailio/project_styleguide/templates/patterns/pages/content_page/content_page.html

```diff
@@ -0,0 +1,22 @@
+{% extends "patterns/base_page.html" %}
+{% load wagtailcore_tags wagtailimages_tags static %}
+
+{% block extra_css %}
+{% endblock %}
+
+{% block body_class %}{% endblock %}
+
+{% block content %}
+    {# title #}
```

Comment:
Note: A comment should provide additional context for a developer to understand what's happening, I think `{% if page.title %} provides this context already, what do you think?

Reply:
Please note this applies to other comments below, too.

Reply:
I agree! Good shout, thanks!

Reply:
updated in ae8d891